src/SignhostAPIClient/Rest/DataObjects/Ideal.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Signhost.APIClient.Rest.DataObjects
+{
+	public class Ideal : Verification
```

Comment:
Inheritance should be placed on a new line like this:
public class Ideal
    : Verification
{
}